plugins/com.google.cloud.tools.eclipse.googleapis/src/com/google/cloud/tools/eclipse/googleapis/internal/DefaultAccountProvider.java

```diff
@@ -224,7 +225,9 @@ protected String getRefreshTokenFromCredentialFile() {
       JsonElement root = JsonParser.parseReader(reader);
       return root.getAsJsonObject().get("refresh_token").getAsString();
     } catch (IOException ex) {
-      LOGGER.log(Level.SEVERE, "Could not open credentials file");
+      LOGGER.log(Level.SEVERE, "Could not open credentials file", ex);
+    } catch (IllegalArgumentException ex) {
```

Comment:
`IllegalStateException` is what I was experiencing. This wouldn't catch it.

Reply:
fixed